### PR TITLE
Added goodtables

### PR DESCRIPTION
**Pull request by roll**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Based on https://github.com/conda-forge/datapackage-feedstock/blob/master/recipe/meta.yaml

### CODE REVIEW

File: recipes/goodtables/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - goodtables = goodtables.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six>=1.9
```
Comment (xhochy) [inline recipes/goodtables/meta.yaml]: Please leave a space between name and version spec for all dependencies.
```suggestion
    - six >=1.9
```

File: recipes/goodtables/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e363d0d2762f5ac48b5f408bfe45c0de1ce8bad908cff42464bb11616ad40e55
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - goodtables = goodtables.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (xhochy) [inline recipes/goodtables/meta.yaml]: No need for the additional arguments, they are all supplied by default now.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv"
```

**Comment by xhochy** [comment]
The builds fail due to https://github.com/conda-forge/simpleeval-feedstock/issues/1

**Comment by roll** [comment]
@xhochy 
Hi, thanks for the review. I've updated the requirements and the script. But I'm not sure how I can fix the `noarch` issue. I guess the only one dependency requiring compiling is `cchardet` via `tabulator` - https://github.com/conda-forge/tabulator-feedstock/blob/master/recipe/meta.yaml. And it seems `tabulator` has the same meta

**Comment by xhochy** [comment]
@roll I made a PR to `simpleeval` to fix the problem: https://github.com/conda-forge/simpleeval-feedstock/pull/2 Once this is merged, we can retrigger the builds here.

**Comment by roll** [comment]
@xhochy
Great! Thanks for your help.

**Comment by ocefpaf** [comment]
Closing and opening to toggle the CIs.

**Comment by roll** [comment]
Now we fail with:

```
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/environ.py", line 757, in get_install_actions
    actions = install_actions(prefix, index, specs, force=True)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/plan.py", line 473, in install_actions
    txn = solver.solve_for_transaction(prune=prune, ignore_pinned=not pinned)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/core/solve.py", line 107, in solve_for_transaction
    force_remove, force_reinstall)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/core/solve.py", line 145, in solve_for_diff
    force_remove)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/core/solve.py", line 242, in solve_final_state
    ssc = self._run_sat(ssc)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/core/solve.py", line 475, in _run_sat
    conflicting_specs = ssc.r.get_conflicting_specs(tuple(final_environment_specs))
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/resolve.py", line 852, in get_conflicting_specs
    reduced_index = self.get_reduced_index(specs)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/resolve.py", line 356, in get_reduced_index
    specs, features = self.verify_specs(specs)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda/resolve.py", line 244, in verify_specs
    raise ResolvePackageNotFound(bad_deps)
conda.exceptions.ResolvePackageNotFound: 
  - simpleeval[version='>=0.9.8'] -> libgcc-ng[version='>=7.3.0']

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".ci_support/build_all.py", line 131, in <module>
    build_all(args.recipes_dir, args.arch)
  File ".ci_support/build_all.py", line 68, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File ".ci_support/build_all.py", line 121, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/build.py", line 2313, in build_tree
    notest=notest,
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/build.py", line 1377, in build
    create_build_envs(top_level_pkg, notest)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/build.py", line 1279, in create_build_envs
    raise e
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/build.py", line 1269, in create_build_envs
    channel_urls=tuple(m.config.channel_urls))
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/environ.py", line 759, in get_install_actions
    raise DependencyNeedsBuildingError(exc, subdir=subdir)
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform osx-64: {"libgcc-ng[version='>=7.3.0']"}
##[error]Bash exited with code '1'.
##[section]Finishing: CmdLine
```

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=44651

**Comment by ocefpaf** [comment]
I just moved the faulty noarch `simpleeval` to broken, see https://anaconda.org/conda-forge/simpleeval/files?version=0.9.8&channel=broken

Let's wai ~20 min for the index update and re-start the CIs here.

**Comment by ocefpaf** [comment]
Looks like there is another problem here :-/
I'll try to get a Windows machine to debug this. Hold on...

File: recipes/goodtables/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - python
+    - six >=1.9
+    - click >=6.6
+    - click-default-group
+    - requests >=2.10
+    - simpleeval >=0.9
+    - statistics >=1.0
+    - tabulator >=1.20
+    - tableschema >=1.0.3
+    - datapackage >=1.2
```
Comment (ocefpaf) [inline recipes/goodtables/meta.yaml]: Trying to debug this on my Windows machine I was not able to install an env with these dependencies b/c of:

```yaml
  - statistics >=1.0
  - tabulator >=1.20
  - tableschema >=1.0.3
  - datapackage >=1.2
```

I'm checking the package now to see what is going on...

Comment (ocefpaf) [reply recipes/goodtables/meta.yaml]: The first one, statistic, is a python 2 only package!

Comment (ocefpaf) [reply recipes/goodtables/meta.yaml]: Looks like many of those depend on `openpyxl` and that one may need to be rebuilt. Working on it...

**Comment by ocefpaf** [comment]
Closing and re-opening to toggle the CIs.

**Comment by ocefpaf** [comment]
Passing now. The extra OS X failing there looks like an Azure artifact. @xhochy, you are the original reviewer here. Do you want to take a second look and merge?

**Comment by roll** [comment]
@ocefpaf 
Actually, you made it! It was kinda weird double run on OsX causing an error. But now it passes. Thank a lot for sorting it out!

**Pull request merged**